testing/cucumber.yml

```diff
@@ -1,7 +1,9 @@
 passing: --tags 'not @failing and not @future_release'
 
-html: --format html --out artifacts/reports/report.html
-junit: --format junit --out artifacts/reports
-json: --format json --out artifacts/reports/report.json
+<% base_path = "artifacts/#{ENV['BROWSER'])}/#{ENV.fetch('BROWSERSTACK_CONFIGURATION_OPTIONS', 'other')}" %>
```

Comment:
Sorry I know you're going to hate me. But we probably need to set the conditional here too

```
base_path = 
  if ENV["BROWSERSTACK"] == "true"
    "artifacts/#{ENV['BROWSER']}/#{ENV['BROWSERSTACK_CONFIGURATION_OPTIONS']}"
  else
    "artifacts/#{ENV['BROWSER']}/other"
  end
```

Reply:
Done